test2.js

```diff
@@ -0,0 +1,19 @@
+var foo = '' ;
+
+if(foo  === "bar") {
+  console.log( 'hey' )
+}
+
+
+var foo88 = '' ;
+
+if(foo88  === "b88") {
+  console.log( 'he88' );
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.dev.codacy.org/gh/machadoit-test-org/eslint-example/pullRequest?prid=29725)
```suggestion
  console.log( "he88" );
```